src/main/java/com/example/Game.java

```diff
@@ -3,88 +3,30 @@
 import java.util.InputMismatchException;
 import java.util.Scanner;
 
-class Game {
+public class Game {
     private final Board board;
     private final Player player1;
     private final Player player2;
     private final Scanner scanner = new Scanner(System.in);
 
     public Game() {
-        System.out.println("Bored!Let's play a Game!!\uD83C\uDFB2");
-        System.out.println("Enter the grid size: (3 for 3x3, 4 for 4x4 etc)");
-        int size;
-        while (true)
-        {
-            try{
-                size = scanner.nextInt();
-                scanner.nextLine();
-                if(size>=3)
-                {
-                    break;
-                }
-                else {
-                    System.out.println("Grid size must be 3 or larger than 3");
-                }
-            }
-            catch (InputMismatchException e)
-            {
-                System.out.println("Invalid input! Please enter valid number");
-            }
-        }
-        board = new Board(size);
-
-        System.out.println("So, tell me what's your name?");
-        String playerName;
-        while (true)
-        {
-            playerName = scanner.nextLine();
-            if(!playerName.isEmpty())
-            {
-                break;
-            }
-            else {
-                System.out.println("Name cannot be empty. Please enter a valid name.");
-            }
-        }
+        System.out.println("Bored! Let's play a Game!! \uD83C\uDFB2");
```

Comment:
This things should not be inside constructor,
You should only use constructor to initialize things,
And have separate methods to handle such things.